contracts/ChildChainGaugeInjectorV2.sol

```diff
@@ -554,7 +554,8 @@ KeeperCompatibleInterface
 
 /**
  * @notice Sets the keeper addresses
- * @param keeperAddresses The array of addresses of the keeper contracts, the 0 address anywhere in this list is a wildcard, all addresses can keep
+ * @param keeperAddresses The array of addresses of the keeper contracts
+ * @dev Setting the keeper to address(0) will make `performUpkeep` permissionless
```

Comment:
minor comment: usually `@dev` tag it is just below the `@notice` one instead than below `@param`